ocaml-lsp-server/test/e2e-new/semantic_hl_data.ml

```diff
@@ -0,0 +1,95 @@
+let src0 =
+  {|open Import
```

Comment:
Could you just write some dummy code here instead? E.g. create some simple source that uses one of every syntactical construct. The problem with reusing actual source code here is that you will trip up grep for everyone.